downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -93,6 +93,29 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Introduction of the `int:range` lang library function
+
+The `int:range` lang library function returns an iterable object, which iterates over a range of integers.
+
+The function call `int:range(start, end, step)` will return an iterable object `S` that iterates over a range of integers from `start` (inclusive) to `end` (exclusive), with `step` being the difference successive integers.
```

Comment:
```suggestion
The `int:range(start, end, step)` function call will return an `S` iterable object, which iterates over a range of integers from `start` (inclusive) to `end` (exclusive) with `step` being the difference successive integers.
```

Reply:
Does 

```
an `S` iterable object
```

read right?

It's the name of the object. I'd rather use 

```
an iterable object `S` 
```

or

```
an iterable object (`S`) 
```

Reply:
I think `an iterable object (`S`) ` will be more clear, shall I change to that?